plugins/com.google.cloud.tools.eclipse.sdk.ui/src/com/google/cloud/tools/eclipse/sdk/ui/preferences/CloudSdkPrompter.java

```diff
@@ -95,13 +92,13 @@ public static File getCloudSdkLocation(Shell shell) {
    * @return the Cloud SDK location, or {@code null} if unspecified
    */
   public static File getCloudSdkLocation(IShellProvider shellProvider) {
-    CloudSdkProvider cloudSdkProvider = new CloudSdkProvider(null);
-    File location = cloudSdkProvider.getCloudSdkLocation();
+    CloudSdk cloudSdk = new CloudSdkProvider().getCloudSdk();
+    File location = cloudSdk.getJavaAppEngineSdkPath().toFile();
```

Comment:
nope, same problem (unless you just need to push the changes I'm not seeing here)